MobiFlight/Config/Config.cs

```diff
@@ -216,5 +218,22 @@ public Config FromInternal(String value, bool throwException = false)
             }
             return this;
         }
+
+        public static Config LoadFromFile(string fileName)
+        {
+            TextReader textReader = new StreamReader(fileName);
+            XmlSerializer serializer = new XmlSerializer(typeof(Config));
+            var config = (Config)serializer.Deserialize(textReader);
+            textReader.Close();
+            return config;
+        }
+
+        public void SaveToFile(string fileName)
+        {
+            XmlSerializer serializer = new XmlSerializer(typeof(MobiFlight.Config.Config));
+            TextWriter textWriter = new StreamWriter(fileName);
+            serializer.Serialize(textWriter, this);
+            textWriter.Close();
+        }
```

Comment:
Should there be any exception handling around any of this?

Reply:
there was none so far

Reply:
Odd.